bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -1121,7 +1173,7 @@ void ReplicaImp::onCommitVerifyCombinedSigResult(SeqNum seqNumber, ViewNum v, bo
     ps_->setCommitFullMsgInSeqNumWindow(seqNumber, commitFull);
     ps_->endWriteTran();
   }
-
+  LOG_INFO(GL, "Commit path analysis: request commited, proceeding to try to execute");
```

Comment:
Are we using `INFO` to analyze a current bug? Otherwise this will spam the logs, correct?

Reply:
More general, should we use `INFO` to log messages in the happy path? I'm afraid that this will just clutter logs.

Reply:
It's not for a specific bug.
It's to mark the end of the Consensus and to be able to measure its duration. we have performance issues and we are blind at the analysis of the Consensus.
We just don't know what takes so long

Reply:
It will produce a lot of noize.
And I tried to minimize it to significant events. but I don't have a better alternative.
If we'll see it's redundant, I'll reduce its level.

Reply:
the "Commit path analysis" prefix is for easy filtering

Reply:
I think that most of recent logs will be moved to DEBUG after system will become stable

Reply:
This must be removed ASAP.